api/src/lib/db.ts

```diff
@@ -47,16 +46,16 @@ async function createPrismaClient() {
     datasourceUrl = await ssmAuthForURL(process.env.DATABASE_URL)
   }
 
-  db = new PrismaClient({
+  const db = new PrismaClient({
     log: emitLogLevels(['info', 'warn', 'error']),
     datasourceUrl: datasourceUrl,
   })
-}
-
-createPrismaClient().then(() => {
   handlePrismaLogging({
     db,
     logger,
     logLevels: ['info', 'warn', 'error'],
   })
-})
+  return db
+}
+
+export const db: PrismaClient = await createPrismaClient()
```

Comment:
Back when we were first setting this up, I had issues implementing the top-level `await` in this manner. Fine with trying this out, but YMMV.

Reply:
ah missed this comment! Unfortunately it doesn't work with our commonjs setup so we need some other way to get this working.